docs/weekly-questions/joes-water-corporation.md

```diff
@@ -117,129 +117,99 @@ There is only one store, where we have a higher price than our competitor. Thus
 When we apply the discount, our price decreases to $5$. However, as our competitor wins in the case of a tie, we still make $0$ profit.
 Therefore, as our profit did not change, the output is `NO`.
 
-<!--
```

Comment:
Please restore the comments. We don't want the model solution to be visible during the meeting.